src/components/settings/forms/ExtraOptions.vue

```diff
@@ -0,0 +1,307 @@
+<template>
+  <section class="sector-extra-options-form">
+    <section class="switchs">
+      <h2 class="switchs__title">
+        {{ $t('sector.additional_options.title') }}
+      </h2>
+      <UnnnicSwitch
+        v-model="sector.can_trigger_flows"
+        :textRight="
+          sector.can_trigger_flows
+            ? $t('sector.additional_options.template_message.switch_active')
+            : $t('sector.additional_options.template_message.switch_disabled')
+        "
+      />
+      <section class="switchs__container">
+        <UnnnicSwitch
+          v-model="sector.sign_messages"
+          :textRight="
+            sector.sign_messages
```

Comment:
Add this snippet to a variable (computed) or a function to facilitate future maintenance